source/ShipJumpNavigation.cpp

```diff
@@ -128,8 +126,6 @@ double ShipJumpNavigation::JumpDriveFuel(double distance) const
 // If no jump method is possible, returns JumpType::None with a jump cost of 0.
 pair<JumpType, double> ShipJumpNavigation::GetCheapestJumpType(const System *destination) const
 {
-	if(!ship->GetSystem())
```

Comment:
here too?

Reply:
This one is useless because the function it calls checks for this